src/plugins/management/public/legacy/sections_register.js

```diff
@@ -20,6 +20,33 @@
 import { LegacyManagementSection } from './section';
 import { i18n } from '@kbn/i18n';
 
+export const sections = [
+  {
+    id: 'ingest',
```

Comment:
Great idea! Will do.

Reply:
I'm hoping we'll change how sections are referenced in the near future. I'm 👎 on looking items up by `id` unless necessary and this case is not necessary. We should be exporting the sections as public APIs of the plugins that create them. The current usage creates an implicit dependency and the potential for runtime errors instead of compile time errors.